package.json

```diff
@@ -8,11 +8,12 @@
 	},
 	"types": "./lib/index.d.ts",
 	"dependencies": {
-		"ardrive-core-js": "1.15.1",
+		"ardrive-core-js": "1.16.0-alpha-2",
```

Comment:
Don't forget to update this 😬

Reply:
Thanks for the reminder. I'll update this just after CORE is released